libr/flag/flag.c

```diff
@@ -502,6 +502,36 @@ R_API RFlagItem *r_flag_get_i(RFlag *f, ut64 off) {
 	return list ? evalFlag (f, r_list_get_top (list)) : NULL;
 }
 
+/* return the first flag that matches an offset ordered by the order of
+ * operands to the function.
+ * XXX this is kind of super ugly! */
```

Comment:
what's exactly the reason?

Reply:
do you still think it's ugly? :) if so, please tell us we can try to find some other solutions!